change/@itwin-reports-config-widget-react-08145948-89dc-4064-a856-f17f7fbe3c64.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Is it a patch if major version changed of a dependency?

Reply:
There is no introduction of new features or breaking changes. I believe a patch is appropriate.

Reply:
@aruniverse  ?

Reply:
Pre-release (`0.y.z`) require bumping `y` (minor) on breaking changes. Bumping the imodels-client major version might be a breaking change/

Reply:
Changed it to minor.